mlir/include/mlir/Dialect/Affine/IR/AffineOps.h

```diff
@@ -16,6 +16,7 @@
 
 #include "mlir/Dialect/Affine/IR/AffineMemoryOpInterfaces.h"
 #include "mlir/Dialect/Arith/IR/Arith.h"
+#include "mlir/Dialect/UB/IR/UBOps.h"
```

Comment:
Is this necessary?

Reply:
Thank you, I was sure I needed it, but you're right!